sdk/order-planner-service/tsconfig.build.json

```diff
@@ -0,0 +1,20 @@
+{
+  // by using a separate tsconfig for build, we can set main tsconfig to
+  // include tests and use module paths in tests but build without tests
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "paths": {
+      "~orderplannerservice": ["src/index.ts"],
+      "~orderplannerservice/*": ["./src/*"],
+      "~orderplannercommon": ["../order-planner-common/src/index.ts"],
+      "~orderplannercommon/*": ["../order-planner-common/src/*"],
+      "~protocolplugins": ["../protocol-plugins/src/index.ts"],
+      "~protocolplugins/*": ["../protocol-plugins/src/*"],
+      "~sdk-common": ["../sdk-common/src/index.ts"],
+      "~sdk-common/*": ["../sdk-common/src/*"],
+      "~swap-common/*": ["../swap-common/src/*"],
+      "~deployment-types": ["../../packages/deployment-types/src/index.ts"]
```

Comment:
shouldn't those be referencing to node_modules instead of relative path

Reply:
No, it is just that after some iterations we started using relative paths. Then you tried to use node_modules, but not all were changed. I'll change those to use the first level `node_modules`, but not a second level as I think it would be too much

Reply:
ok cool, we'll have another look at this later